pkg/api/v1beta2/types.go

```diff
@@ -1665,8 +1665,8 @@ const (
 )
 
 type ComponentCondition struct {
-	Type    ComponentConditionType `json:"type"`
-	Status  ConditionStatus        `json:"status"`
+	Type    ComponentConditionType `json:"type" description:"the type of condition"`
```

Comment:
Sorry, that should have been on Status.

Here, copy the text from v1beta3 that indicates that Healthy is the only condition type currently